guilford_court_commitment.yaml

```diff
@@ -0,0 +1,408 @@
+entityDefinitions:
+
+
+
+  offenders:
+    fqn: "general.person"
+    entitySetName: "GuilfordOffenders"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        transforms:
```

Comment:
This pk must match the other DPS flights because they are all merging - see the documentation on the parent wiki page for DPS data - https://wiki.openlattice.com/pages/viewpage.action?pageId=59280140

OFFENDER NC DOC ID NUMBER with 7 digits